.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mdf"]
+	path = mdf
+	url = https://github.com/edwardcwang/plsi-mdf.git
```

Comment:
Let's discuss this in person then - I think in the interest of re-use, I'd like to have the macro description format be an independent library so I can use it for other passes and projects.